app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="container-fluid w-50 d-flex flex-column align-items-center" >
+  <h1>Shopping List</h1>
+  <h2>Recipe <%= @recipe.name %></h2>
+<div class="container-fluid w-100 d-flex justify-content-between align-items-center my-4">
+  <p>Amount of Food Items to buy: <%= @total_missing_food %> </p>
+  <p>Total Value: <%= @total_price %>$</p>
+</div>
+<table class="table table-bordered table-striped">
+  <thead>
+    <tr>
+      <th scope="col">Food </th>
+      <th>Recipe Quantity </th>
+      <th>User Quantity </th>
+      <th>Unit price </th>
+      <th>Actions </th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @missing_food.each do |food| %>
+      <tr>
+        <td> <%= food.food.name %></td>
+        <td> <%= food.quantity %></td>
+        <td> <%= food.food.quantity || 0 %></td>
+        <td> $ <%= food.food.price%></td>
+        <td><%= button_to 'delete', recipe_food_path(food.id), method: :delete, class:"btn btn-warning"%></td>
+      </tr>
+    <%end%>
+  </tbody>
+</table>
```

Comment:
- [ ] When a user clicks on generate list button, the page only shows the title of a recipe with empty data, even if the food was added. This is against the project requirements.

![food](https://user-images.githubusercontent.com/53124548/201416002-098e05fb-faa3-4f00-aa4f-9eeeb040f27b.png)

Reply:
@elisha2kyakpo1 could you please check it again?
Its working properly for me 
<img width="1440" alt="Screen Shot 2022-11-12 at 12 12 06 AM" src="https://user-images.githubusercontent.com/53530780/201417789-3034a2a3-cfde-4cef-9c4a-72afd206acbb.png">
<img width="1439" alt="Screen Shot 2022-11-12 at 12 11 58 AM" src="https://user-images.githubusercontent.com/53530780/201417800-2e3eeeec-73fa-4009-87ec-7308b51e70bf.png">